_maps/map_files/BallMap/summer_ball.dmm

```diff
@@ -0,0 +1,3184 @@
+"zx" = (/obj/structure/table/greyscale,/obj/item/reagent_containers/food/drinks/bottle/vodka/badminka{pixel_x = -7; pixel_y = 8},/obj/item/reagent_containers/food/drinks/bottle/vodka/badminka{pixel_x = 7; pixel_y = 8},/obj/structure/sign/poster/official/high_class_martini{pixel_x = 0; pixel_y = -32},/turf/open/floor/wood,/area/bridge/meeting_room/council)
+"zy" = (/obj/structure/chair/comfy{color = "#596479"; dir = 4},/turf/open/floor/plasteel/dark,/area/bridge/meeting_room/council)
+"zz" = (/obj/structure/table/wood/fancy/royalblue,/turf/open/floor/plasteel/dark,/area/bridge/meeting_room/council)
+"zA" = (/obj/structure/table/wood/fancy/royalblue,/obj/item/toy/cards/deck/cas,/obj/item/toy/cards/deck/cas/black{pixel_x = -5; pixel_y = 4},/turf/open/floor/plasteel/dark,/area/bridge/meeting_room/council)
+"zB" = (/obj/machinery/door/airlock/medical/glass{name = "Emergency Medbay"},/turf/open/floor/plasteel/white,/area/medical/medbay/lobby)
+"zC" = (/obj/structure/chair/comfy{color = "#596479"; dir = 8},/turf/open/floor/plasteel/dark,/area/bridge/meeting_room/council)
+"zD" = (/obj/structure/statue/diamond/ai2{desc = "This is a very expensive diamond statue of an AI that isn't in the casino for the safety of everyone else involved."; name = "C.A.S.I.N.O."},/turf/open/floor/plasteel/dark,/area/bridge/meeting_room/council)
+"zE" = (/turf/open/floor/plasteel/dark/side{dir = 4},/area/crew_quarters/abandoned_gambling_den)
+"zF" = (/obj/structure/sink{dir = 1; pixel_y = -9},/turf/open/floor/plasteel/showroomfloor,/area/crew_quarters/dorms)
+"zG" = (/obj/structure/table/wood/fancy/royalblue,/obj/item/toy/cards/deck/cas{pixel_y = 4},/obj/item/toy/cards/deck/cas/black{pixel_x = 5; pixel_y = 8},/turf/open/floor/plasteel/dark,/area/bridge/meeting_room/council)
+"zH" = (/obj/structure/chair/comfy{color = "#596479"; dir = 1},/turf/open/floor/plasteel/dark,/area/bridge/meeting_room/council)
+"zI" = (/obj/structure/sign/poster/official/soft_cap_pop_art{pixel_y = -32},/turf/open/floor/carpet/royalblue,/area/bridge/meeting_room/council)
+"zJ" = (/turf/open/floor/plasteel/white,/area/crew_quarters/abandoned_gambling_den)
+"zK" = (/obj/structure/closet/crate/freezer,/obj/item/storage/cans/sixbeer,/turf/open/floor/wood,/area/awaymission/beach)
+"zL" = (/obj/structure/table/reinforced,/obj/effect/turf_decal/trimline/blue/filled/line{dir = 1},/turf/open/floor/plasteel/white,/area/medical/medbay/lobby)
+"zM" = (/obj/mafia_game_signup,/turf/open/floor/plasteel/grimy,/area/crew_quarters/abandoned_gambling_den)
+"zN" = (/obj/structure/chair/comfy/brown{color = "#66b266"; desc = "Not to be confused with messaging other people."; dir = 2; max_integrity = 9999; name = "DM Chair"},/obj/item/clothing/head/wizard{desc = "DnD swag, 20 d20s in my dicebag."; name = "Hat of Dungeon Mastery +3"; pixel_y = 12},/turf/open/floor/carpet/green,/area/crew_quarters/abandoned_gambling_den)
```

Comment:
you might wanna use a fake wizard hat here; real ones have armor

Reply:
given that you can buy armored clothing from the jobdrobes, I don't think it's really that dangerous to include a wizard hat